contributing/community.ipynb

```diff
@@ -0,0 +1,68 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9cbbef13-a05c-4169-9627-e54a43a11ee1",
+   "metadata": {},
+   "source": [
+    "# Research Coding Community (RCC)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "629d3135-1e5a-41dd-bae3-638d4af4e899",
+   "metadata": {},
+   "source": [
+    "The Research Coding Community (RCC) is a network brought together for everyone across the university who writes code in support of their research. Having just completed your CfRR course, signing up to the RCC is the next step in connecting with peers who share your interests and challenges in computational research.\n",
+    "\n",
+    "Currently hosted online via Microsoft Teams, the RCC offers a friendly virtual space where you can ask questions, share tips, and discover collaborators. We’re committed to growing the community by co-creating a calendar of activities and events shaped by what members find most valuable. Potential offerings include:\n",
+    "\n",
+    "- **Seminars** where members present their own projects or dive into industry best practices.\n",
+    "- **Hands-on workshops** on topics like version control, reproducible pipelines, or emerging programming languages.\n",
+    "- **Hack days**, one-day mini-hackathons, to tackle real problems in small teams.\n",
+    "- **Co-working sessions** for group coding, pair programming, or just having company while you work.\n",
+    "- **Social events** to unwind, network, and celebrate our collective wins.\n",
+    "  \n",
+    "Whether you’re troubleshooting a tricky script, learning a new tool, or just want to meet others applying code in novel ways, the RCC is here to help you get to know, and code with, your research community. We’d love to have you on board!"
```

Comment:
Again, it might be worth tweaking this slightly. How about the following?

> Whether you’re troubleshooting a tricky script, learning a new tool, or wanting to meet others applying code in novel ways, the RCC is here to help you get to know, and code with, the wider community. We’d love to have you on board!